cordova-plugin-ms-adal/cordova-plugin-ms-adal.d.ts

```diff
@@ -0,0 +1,214 @@
+// Type definitions for Active Directory Authentication Library plugin for Apache Cordova
+// Project: https://github.com/AzureAD/azure-activedirectory-library-for-cordova
+// Definitions by: Kai Walter <https://github.com/KaiWalter>
+// Definitions: https://github.com/KaiWalter/DefinitelyTyped
+
+declare namespace Microsoft {
+
+    namespace ADAL {
+
+        interface IUserInfo {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files
> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.